src/main/resources/templates/languages/cy/non-strategic/fftLrWeeklyHearingList.json

```diff
@@ -0,0 +1,12 @@
+{
+  "listName": "Rhestr o Wrandawiadau Wythnosol Tribiwnlys Haen Gyntaf (Tribiwnlys Cofrestru Tir)",
+  "contactMessage": "Gall aelodau o’r cyhoedd sy’n dymuno arsylwi gwrandawiad neu gynrychiolwyr y cyfryngau ymuno ag unrhyw wrandawiad dros y ffôn neu drwy fideo o bell ar gais tra’u bod yn cael eu cynnal drwy anfon e-bost ymlaen llaw at y tribiwnlys yn [insert office email] gyda’r manylion canlynol yn y llinell bwnc “CAIS [ARSYLLWR/CYFRYNGAU] – [cyfeirnod yr achos] – [dyddiad y gwrandawiad] (angen cynnwys unrhyw wybodaeth arall sy’n ofynnol gan y tribiwnlys)” a gwneir trefniadau priodol i ganiatáu mynediad lle bo hynny’n rhesymol ymarferol.",
+  "tableHeaders": [
+    "Dyddiad",
+    "Amser y gwrandawiad",
+    "Enw’r achos",
+    "Cyfeirnod yr achos",
+    "Judge",
+    "Lleoliad/Platfform"
```

Comment:
The word Platform not translated to Welsh

Reply:
as discussed, platform has double f in welsh